mappings/net/minecraft/village/VillagerProfession.mapping

```diff
@@ -1,5 +1,51 @@
 CLASS net/minecraft/unmapped/C_yhqxfbuc net/minecraft/village/VillagerProfession
 	FIELD f_acjlkvad secondaryJobSites Lcom/google/common/collect/ImmutableSet;
-	FIELD f_ayvuterw gatherableItems Lcom/google/common/collect/ImmutableSet;
+	FIELD f_ayvuterw requestedItems Lcom/google/common/collect/ImmutableSet;
+	FIELD f_kmphlego potentialJobSites Ljava/util/function/Predicate;
+	FIELD f_obosegzt primaryJobSites Ljava/util/function/Predicate;
 	FIELD f_otcpaipa workSound Lnet/minecraft/unmapped/C_avavozay;
-	FIELD f_whefzzlg id Ljava/lang/String;
+	FIELD f_vqqijdeq ANY_JOB_SITE Ljava/util/function/Predicate;
+	FIELD f_whefzzlg name Ljava/lang/String;
+	METHOD <init> (Ljava/lang/String;Ljava/util/function/Predicate;Ljava/util/function/Predicate;Lcom/google/common/collect/ImmutableSet;Lcom/google/common/collect/ImmutableSet;Lnet/minecraft/unmapped/C_avavozay;)V
+		ARG 1 name
+		ARG 2 primaryJobSites
+		ARG 3 potentialJobSites
+		ARG 4 requestedItems
+		ARG 5 secondaryJobSites
+		ARG 6 workSound
+	METHOD m_afajbkbx register (Ljava/lang/String;Lnet/minecraft/unmapped/C_xhhleach;Lnet/minecraft/unmapped/C_avavozay;)Lnet/minecraft/unmapped/C_yhqxfbuc;
+		ARG 0 id
+		ARG 1 key
+		ARG 2 workSound
+	METHOD m_arfdbrqv heldJobSite ()Ljava/util/function/Predicate;
```

Comment:
Is this a record?

Reply:
yes, so I'm guessing the method doesn't need mapping? Enigma had greyed names that didn't match the fields, and I'm not sure what that meant so I mapped them